<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSysManage</class>
 <widget class="QWidget" name="addSysManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>接口名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="systemName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="deviceType">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>接口类型</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QRadioButton" name="UdpButton">
        <property name="text">
         <string>UDP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UartButton">
        <property name="text">
         <string>UART</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BButton">
        <property name="text">
         <string>1553B</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>UDP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ip地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipAddr"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipPort"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>1553B</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>板 卡 号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="RTADDR">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>模式选择：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>RT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BC</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>RT 地 址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>UART</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>串口号:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>波特率:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>数据位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>停止位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>校验位:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>N</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>147</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UdpButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UartButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UartButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UdpButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UdpButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UartButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BButton</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
